common/scala/build.gradle

```diff
@@ -32,3 +32,7 @@ dependencies {
     compile 'org.apache.kafka:kafka-clients:0.10.0.0'
     compile 'org.apache.httpcomponents:httpclient:4.4.1'
 }
+
+tasks.withType(ScalaCompile) {
+    scalaCompileOptions.additionalParameters = ['-feature', '-deprecation', '-Xfatal-warnings', '-Ywarn-unused-import']
```

Comment:
can these be consolidated to one setting file (rather than repeat in each scala project)?